packages/api/models/CloudServiceProvider.ts

```diff
@@ -0,0 +1,4 @@
+export const enum CloudServiceProvider {
+  AmazonWebServices = "aws",
+  MicrosoftAzure = "azure",
+}
```

Comment:
Enum values should usually be ALL_UPPER_CASE_WITH_UNDERSCORES. I think these names get a little long for that. Thoughts on `CloudServiceProvider.AWS` or CloudServiceProvider.AZURE`?

Reply:
Noticed we're not always consistent between `enum` and `const enum`. Can we add guidance on when to pick one or another to our development docs?

Reply:
I think this would help differentiate the enums and should be applied to `ProvisioningStatus` as well.

Reply:
Changed `CloudServiceProvider` and `ProvisioningStatus` in 5026b90f6e1cd6004a8715cc1647ed180a72bb3e.

Reply:
Investigated `enum` vs `const enum`.  Removed `const` from enum declarations in 37cc2026f05a428af571abe399bbaffbdf4ff2bd.